typescript/package.json

```diff
@@ -0,0 +1,62 @@
+{
+	"name": "<%= moduleName %>",
+	"version": "0.0.0",
+	"description": "",
+	"license": "MIT",
+	"repository": "<%= githubUsername %>/<%= moduleName %>",
+	"author": {
+		"name": "<%= name %>",
+		"email": "<%= email %>",
+		"url": "<%= website %>"
+	},
+	"engines": {
+		"node": ">=6"
+	},
+	"scripts": {
+		"build": "del dist && tsc",
+		"prepare": "npm run build",
+		"pretest": "npm run build",
+		"test": "xo && ava"
+	},
+	"main": "dist",
+	"types": "dist",
```

Comment:
I personally prefer to put non-standard package.json properties at the end.

Reply:
My idea here is that it nearly does the same as `main` does.